src/mach.rs

```diff
@@ -195,6 +204,7 @@ impl SectionBuilder {
             addr: 0,
             align: 4,
             offset: 0,
+            flags: 2147484672,
```

Comment:
I know this was already present, but could you name this flags value? Also, would it make sense to format it in hexadecimal?